go/webdriver/webdriver_test.go

```diff
@@ -304,8 +304,40 @@ func TestActiveElement(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if id != "input" {
-		t.Fatalf("Expected 'input' but got '%v'", id)
+	if id != "input1" {
+		t.Fatalf("Expected 'input1' but got '%v'", id)
+	}
+}
+
+func TestKeyDownUp(t *testing.T) {
+	ctx := context.Background()
+
+	d, err := CreateSession(ctx, wdAddress(), 3, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer d.Quit(ctx)
+
+	testURL, _ := testURL("webdriver_autofocus.html")
+	if err := d.NavigateTo(ctx, testURL); err != nil {
+		t.Fatal(err)
+	}
+
+	d.KeyDown(ctx, TabKey)
+	d.KeyUp(ctx, TabKey)
```

Comment:
It seems one key fact of these newly added functions is that they could support multiple keys being keydown/keyup at the same time. Could you please improve the test to cover that case as well?

Reply:
Ack. I'm mirroring an internal contribution. Let's follow up with a separate PR (discussed in 1:1).